data/src/main/java/com/example/data/datasources/remote/NetworkModule.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.data.datasources.remote
+
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+class NetworkModule {
+    companion object {
+        lateinit var ApiClient: UserApiService
+        fun apiClient(): UserApiService {
+            return  Retrofit.Builder()
+                    .baseUrl("https://5e510330f2c0d300147c034c.mockapi.io/")
```

Comment:
try to move the base URL to `@String/base_url` or to `Gradle `to make this code more readable